lib/ansible/modules/cloud/azure/azure_rm_resource.py

```diff
@@ -114,6 +114,7 @@
 
 from ansible.module_utils.azure_rm_common import AzureRMModuleBase
 from ansible.module_utils.azure_rm_common_rest import GenericRestClient
+from ansible.module_utils.common.dict_transformations import dict_merge
 from copy import deepcopy
```

Comment:
Doesn't look this this is needed anymore, quick check shows that it was only used in `dict_merge`

Reply:
Fixed